pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -0,0 +1,1166 @@
+        601: ("SYSTEM", "Hardware/firmware interaction; check other errors"),
+        650: ("OUTOF RG", "ADC input is outside of the ADC range"),
+        651: ("NO IRQ", "Microprocessor is not receiving interrupt from ADC"),
+        700: ("EEROM", "Checksum error of EEROM A2U501"),
+        701: ("AMPL CAL", "Checksum error of frequency response correction data"),
+        702: ("ELAP TIM", "Checksum error of elapsed time data"),
+        703: ("AMPL CAL", "Checksum error of frequency response correction data"),
+        704: ("PRESELCT", "Checksum error of customer preselector peak data"),
+        705: ("ROM U306", "Checksum error of program ROM A2U306"),
+        706: ("ROM U307", "Checksum error of program ROM A2U307"),
+        707: ("ROM U308", "Checksum error of program ROM A2U308"),
+        708: ("ROM U309", "Checksum error of program ROM A2U309"),
+        709: ("ROM U310", "Checksum error of program ROM A2U310"),
+        710: ("ROM U311", "Checksum error of program ROM A2U311"),
+        711: ("RAM U303", "Checksum error of system RAM A2U303"),
+        712: ("RAM U302", "Checksum error of system RAM A2U302"),
+        713: ("RAM U301", "Checksum error of system RAM A2U301"),
+        714: ("RAM U300", "Checksum error of system RAM A2U300"),
+        715: ("RAM U305", "Checksum error of system RAM A2U305"),
+        716: ("RAM U304", "Checksum error of system RAM A2U304"),
+        717: ("BAD uP!!", "Microprocessor not fully operational"),
+        718: ("BATTERY?", "Nonvolatile RAM not working; check battery"),
+        750: ("SYSTEM", "Hardware/ firmware interaction; check other errors"),
+        751: ("SYSTEM", "Hardware/firmware interaction; check other errors"),
+        752: ("SYSTEM", "Hardware/firmware interaction; check other errors"),
+        753: ("SYSTEM", "Hardware/firmware interaction; check other errors"),
+        754: ("SYSTEM", "Hardware/firmware interaction; check other errors"),
+        755: ("SYSTEM", "Hardware/firmware interaction; check other errors"),
+        900: ("TG UNLVL", "Tracking generator output is unleveled"),
+        901: ("TGFrqLmt",
+              "Tracking generator output unleveled because START FREQ is set "
+              "below tracking generator frequency limit (300 kHz)"),
+        902: ("BAD NORM",
+              "The state of the stored trace does not match the current state of the analyzer"),
+        903: ("&> DLMT", "Unnormalized trace A is off-screen with trace math or normalization on"),
+        904: (
+            "&> DLMT",
+            "Calibration trace (trace B) is off-screen with trace math or normalization on")
+    }
+
+    def __init__(self, code: int):
+        self.code = int(code)
+        (self.short, self.long) = self.__error_code_list[self.code]
+
+    def __repr__(self):
+        return "ErrorCode(\"" + self.short + " - " + self.long + "\")"
+
+    def __eq__(self, other):
+        return self.code == other.code
+
+
+class HP856Xx(Instrument):
+    """
```

Comment:
Please provide a short description of the device, typically something like "Represents the HP856Xx series oscilloscopes.